apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -276,18 +273,18 @@ export const CssValueInput = ({
         ? String(item.value)
         : toValue(item),
     onInputChange: (inputValue) => {
-      onChange(inputValue ?? unsetValue.value);
+      onChange(inputValue ?? toValue(props.value));
     },
     onItemSelect: (value) => {
-      onChangeComplete(value ?? unsetValue);
+      onChangeComplete(value ?? props.value);
     },
     onItemHighlight: (value) => {
       if (value == null) {
-        onHighlight(unsetValue);
+        onHighlight(props.value);
         return;
       }
       if (value.type !== "intermediate") {
-        onHighlight(value ?? unsetValue);
+        onHighlight(value ?? props.value);
       }
     },
   });
```

Comment:
Below we have code like 
```
isOpen && !menuProps.empty
```

Seems like with new controllable `isOpen` we don't need to check menuProps.empty